mmv1/templates/terraform/examples/cloud_run_service_probes.tf.erb

```diff
@@ -0,0 +1,35 @@
+data "google_project" "project" {
+  provider = google-beta
+}
+
+resource "google_cloud_run_service" "<%= ctx[:primary_resource_id] %>" {
```

Comment:
In `terraform.yaml` you'll need to add an `examples` entry to register this! https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/README.md#terraformyaml-metadata

Reply:
Done.